src/flex-example-app-android.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.example.android.snake</id>
```

Comment:
okay, i will add some instructions on how to change code to get it worked with users accounts